### ISSUE TITLE
Bug: DM header and navigation are not contrast compliant

### ISSUE THREAD

**Issue by nataliafitzgerald** [cfpb/design-manual#539]
In testing out the Lighthouse accessibility tool we realized that the Design Manual header and primary navigation do not have sufficient color contrast. The unselected state for text is set to Gray 60 (not accessible). The unselected state should be set to Gray since this is the lightest Gray we can use on White and remain accessible. This will apply to all pages that use the DOCter template. 

https://cfpb.github.io/design-manual/
<img width="1290" alt="screen shot 2017-10-24 at 5 38 28 pm" src="https://user-images.githubusercontent.com/6887128/31969331-3801036c-b8e2-11e7-8a81-eebc13fa1cc8.png">

https://eregs.github.io/
<img width="1284" alt="screen shot 2017-10-24 at 5 40 02 pm" src="https://user-images.githubusercontent.com/6887128/31969405-7bdcbb12-b8e2-11e7-8351-b0b19043b055.png">

**Comment by nataliafitzgerald** [comment]
## Recommendations

### Header
- CFPB Design Manual text: Change to Gray (#5a5d61)
- Unselected text: Change to Gray (#5a5d61)

| Current  | Recommendation | 
| --- | --- | 
| ![with shaded subsections](https://user-images.githubusercontent.com/6887128/36874524-3debd3c2-1d7a-11e8-9adc-136d914f7817.png) | ![without shaded subsections](https://user-images.githubusercontent.com/6887128/36874523-3dc42f70-1d7a-11e8-92b2-eefafa4edfb0.png) | 

### Footer
- "Open source" and "A note about our standards" - Change to H4
- Paragraph text: Change to 1em (16px)
- Background: Change to Gray 5 (#f7f8f9)
- Add rule line: 1px solid Gray 40 (#b4b5b6)
- Link list: Change text to Medium

| Current  | Recommendation | 
| --- | --- | 
| ![with shaded subsections](https://user-images.githubusercontent.com/6887128/36875597-98ba27b4-1d7e-11e8-9ed5-d5773017b7b2.png) | ![without shaded subsections](https://user-images.githubusercontent.com/6887128/36875581-8dadbe76-1d7e-11e8-84b0-d2bddced9211.png) |

**Comment by marteki** [comment]
Thanks, @nataliafitzgerald! I checked the recommendation using the WAVE Chrome extension.

## Header

The "CFPB Design Manual" text is actually color compliant in its current state, because of its larger text size! If it makes more sense to change its color because the other header unselected text colors are changing, that's cool.

## Footer

Great work figuring out the shifts to make so that the existing hover and focus colors for the links become compliant! And also thank you for bringing the general footer closer to our current design guidelines (re: font weight of link lists and general font size and more).

:+1:

**Comment by nataliafitzgerald** [comment]
@marteki 
Yeah, I don't think it hurts to go a little darker on the "CFPB Design Manual" text. In the well, it looks like the hover text for links is also not contrast compliant (Pacific 60) so we have some work to do to bring our global link styling up to par. I went ahead and created an issue for that broader problem here: https://github.com/cfpb/design-manual/issues/605.
